docs/API.md

```diff
@@ -681,8 +687,23 @@ dropped
 | --------------------- | ----- | ------- | ----- | ------ | -------- | ------ |
 | `SRTO_OHEADBW`        | 1.0.5 | post    | `int` | %      | 25       | 5..100 | 
 
-- Recovery bandwidth overhead above input rate (see `SRTO_INPUTBW`). 
+- Recovery bandwidth overhead above input rate (see `SRTO_INPUTBW`). It is
+effective only if `SRTO_MAXBW` is set to 0.
+
 - *Sender: user configurable, default: 25%.* 
+
+- *NOTE: don't configure it too low and avoid 0 in case when you have
```

Comment:
- "in case when you have" -> "in case you have" or "if you have"
- "avoid `0`" - valid values are "5..100" according to this doc (see above). Therefore, 0 is not possible.